test/unit/scanner/images.test.ts

```diff
@@ -56,32 +56,43 @@ tap.test('constructStaticAnalysisOptions() tests', async (t) => {
   t.deepEqual(options, expectedResult, 'returned options match expectations');
 });
 
-tap.test('extracted image tag tests', async (t) => {
-  t.plan(6);
+tap.test('extracted image tag and name tests', async (t) => {
+  t.plan(14);
 
   const imageWithSha = 'nginx@sha256:1234567890abcdef';
   const imageWithShaResult = scannerImages.getImageParts(imageWithSha);
   t.same(imageWithShaResult.imageTag, '1234567890abcdef', 'image sha is returned');
+  t.same(imageWithShaResult.imageName, 'nginx', 'image name is returned');
```

Comment:
btw, image name tests are below ⬇️ in the `tap.test('extracted image name tests',`

Reply:
oh god